plugins/macro/Macro/Utilities.pm

```diff
@@ -49,7 +49,7 @@ sub ai_isIdle {
 		}
 	}
 	return 1 if (AI::is('macro', 'deal'));
-	return 1 if (AI::is('NPC') && $char->args->waitingForSteps);
+	return 1 if (AI::is('NPC') && $char->args->{waitingForSteps});

```

Comment:
How does this even happen?

* `talknpc` [calls `AI::ai_talkNPC`](https://github.com/OpenKore/openkore/blob/master/src/Commands.pm#L4999)
* `AI::ai_talkNPC` creates an AI sequence with the [args set to a Task::TalkNPC task](https://github.com/OpenKore/openkore/blob/master/src/AI.pm#L648)
* `Task::TalkNPC` [has a `waitingForSteps` method](https://github.com/OpenKore/openkore/blob/master/src/Task/TalkNPC.pm#L797)

So there shouldn't be a bug. If people upgraded only a few openkore files instead of the whole thing, it's okay if it doesn't work, IMO.